lib/config/screen.ex

```diff
@@ -20,6 +20,7 @@ defmodule ScreensConfig.Screen do
           | :solari_large
           | :solari_large_v2
           | :pre_fare_v2
+          | :elevator_v2
```

Comment:
In the places in this file that currently have a list of screen types/app IDs, they are currently alphabetized. Let's keep them in order.